ql/src/java/org/apache/hadoop/hive/ql/security/authorization/command/CommandAuthorizerV2.java

```diff
@@ -121,6 +142,24 @@ private static void addPermanentFunctionEntities(SessionState ss, List<ReadEntit
     return hivePrivobjs;
   }
 
+  private static boolean isDeferredAuthView(Table t){
+    String tableType = t.getTTable().getTableType();
+    boolean isView = false;
+    if (TableType.MATERIALIZED_VIEW.name().equals(tableType) || TableType.VIRTUAL_VIEW.name().equals(tableType)) {
+      isView = true;
+    }
+    if(isView){
+      Map<String, String> params = t.getParameters();
+      if (params != null && params.containsKey("Authorized")) {
+        String authorized = params.get("Authorized");
```

Comment:
Instead of using a "Authorized" keyword, it is less error prone to create a constant and reuse it at all the places.